libphobos/src/std/math.d

```diff
@@ -891,11 +891,11 @@ Lret: {}
                 1.0
             ];
 
-            enum real DP1 =
+            enum real P1 =
                 7.853981633974483067550664827649598009884357452392578125E-1L;
-            enum real DP2 =
+            enum real P2 =
                 2.8605943630549158983813312792950660807511260829685741796657E-18L;
-            enum real DP3 =
+            enum real P3 =
                 2.1679525325309452561992610065108379921905808E-35L;
```

Comment:
@ibuclaw As you probably wrote that code: Is changing the identifiers correct here? The rest of the code needs `P1/P2/P3` constants and `DP1/2/3` is not used anywhere.

Reply:
OK, thanks You guys already did some great work on AArch64 support, there only seem to be few changes left for full support 👍